pyproject.toml

```diff
@@ -40,12 +41,10 @@ open-balpy = "<1"
 open-multicaller = "^0.2.0"
 tomli = "^2.2.1"
 apprise = ">=1.7.5,<2.0.0"
-lyra-v2-action-signing = {path = "third_party/upstream/v2-action-signing-python"}
 open-aea-ledger-ethereum = {path = "third_party/forks/open-aea/plugins/aea-ledger-ethereum"}
 eth-account = "^0.13.5"
 eth-typing = "<=5.2.0"
 web3 = ">=5.0.0"
-derive-action-signing = "^0.0.9"
```

Comment:
I believe we only need the latter, which is already dependency of the derive client. I removed it here as I don't think this is a direct dependency of `capitalisation_station`, and hence avoids version conflicts in future updates.